uw-frame-components/portal/settings/controllers.js

```diff
@@ -20,7 +20,8 @@ define(['angular'], function(angular) {
               });
             return partOne + partTwo;
           } else {
-            return input;
+            return input.charAt(0).toUpperCase()
```

Comment:
Backlogged [MUMUP-2970](https://jira.doit.wisc.edu/jira/browse/MUMUP-2970) for MyUW dev team (low priority)